install/detect-platform.sh

```diff
@@ -12,8 +12,14 @@ echo "${_group}Detecting Docker platform"
 # linux/amd64 by default due to virtualization.
 # See https://github.com/docker/cli/issues/3286 for the Docker bug.
 
+export DOCKER_EXISTS=$(command -v docker)
 export DOCKER_ARCH=$(docker info --format '{{.Architecture}}')
```

Comment:
I would have moved this line after the check to avoid a silent failure

Reply:
Sorry, can you elaborate a bit more - which line number are you proposing to move the `export DOCKER_EXISTS..." line to? Or are you suggesting we move it to a different file?

Reply:
The line DOCKER_ARCH will silently fail if docker does not exist

Reply:
Ahhh, forgot that command substitutions get executed at decl time 🤦. Fixed.